src/QMCHamiltonians/PairCorrEstimator.h

```diff
@@ -40,6 +40,9 @@ class PairCorrEstimator : public OperatorBase
   /* evaluate the pair correlation functions */
   Return_t evaluate(ParticleSet& P) override;
 
+  /// generate the unique pair id from the group ids of particle i and j
+  const int gen_pair_id(const int& ig, const int& jg);
```

Comment:
POD type can just pass value. No need of reference. The function needs to be marked as cost member. Return "const" is not needed and has no effect.